test/langtools/tools/javac/Diagnostics/7039014/T7039014a.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * @test /nodynamiccopyright/
+ * @bug     7039014
+ * @summary Confusing error message for method conflict
+ * @author archiecobbs
+ *
+ * @compile/fail/ref=T7039014a.out -XDrawDiagnostics T7039014a.java
+ */
+public class T7039014a {
```

Comment:
I think that T7039014(a,b,c) should be combined into only one test

Reply:
Good idea, thanks - done.